CHANGES.md

```diff
@@ -260,6 +270,7 @@ To be released.
 [#1289]: https://github.com/planetarium/libplanet/pull/1289
 [#1295]: https://github.com/planetarium/libplanet/issues/1295
 [#1296]: https://github.com/planetarium/libplanet/pull/1296
+[#1301]: https://github.com/planetarium/libplanet/issues/1301
```

Comment:
```suggestion
[#1301]: https://github.com/planetarium/libplanet/issues/1301
[#1305]: https://github.com/planetarium/libplanet/pull/1305
```

Reply:
Yup, this isn't fully ready for deployment. :(